pkg/storage/store.go

```diff
@@ -3429,6 +3429,7 @@ func sendSnapshot(
 	{
 		var ent raftpb.Entry
 		for i := range logEntries {
+			ent.Reset()
```

Comment:
`ent.Unmarshal` resets internally. Doesn't it?

Reply:
No, see the discussion linked to in this PR. Unless I really messed up both the unit test there and my code inspection.

Reply:
Oh, it doesn't! That's surprising. Is that a bug in gogoproto? I'm pretty sure `proto.Unmarshal` clears the destination.